src/pages/resources/migration/ios/migrate-to-5x.md

```diff
@@ -71,3 +71,74 @@ Once the previous command is complete, run `pod install` or `pod update` to upda
 If you are using Swift Package Manger (SPM) for managing your app dependencies, you can now include the Experience Platform 5.x SDKs either through Xcode UI, or by declaring them as dependencies in the Package.swift project file. For more details, follow the guide for [managing dependencies using Swift Package Manager](../../manage-spm-dependencies.md).
 
 ## Update outdated API references
+
+When updating to the Experience Platform 5.x SDKs, please take note of the following updates for API references.
+
+#### Edge Bridge
+
+As of version 5.0.0 of AEP Edge Bridge for iOS, the following table lists the mapping of the `trackAction` and `trackState` parameters to the "data" node of the Experience Event sent to the Edge Network. Adobe Experience Platform Edge Network automatically maps these data variables into Adobe Analytics without additional server-side configuration. If you are currently using Edge Bridge version 4.x and you are mapping data to XDM in your datastream, adjustments will be necessary when adopting version 5.0.0.
+
+| Data | Key path in the network request (v4.x) | Key path in the network request (v5+) | Description |
+| --- | --- | --- | --- |
+| Action | `data.action` | `data.__adobe.analytics.linkName` | As of v5, the field `data.__adobe.analytics.linkType` with value `lnk_o` is also automatically included. |
+| State | `data.state` | `data.__adobe.analytics.pageName` | |
+| Context data | `data.contextdata` | `data.__adobe.analytics.contextData` | Context data is a map which includes the custom keys and values specified in the `trackAction` and `trackState` API calls. |
+| Context data prefixed with "&&" | `data.contextdata`| `data.__adobe.analytics` | Before v5, there was no special handling of context data prefixed with "&&".  <br/> <br/> As of v5, context data keys prefixed with `&&` **must** be known to Analytics and are case sensitive. When mapped to the event, the key's name does **not** include the "&&" prefix. For example, "&&products" is sent as `data.__adobe.analytics.products`.|
```

Comment:
```
As of v5, context data keys prefixed with && must be known to Analytics and are case sensitive. When mapped to the event, the key's name does not include the "&&" prefix. For example, "&&products" is sent as data.__adobe.analytics.products.
```

I feel it is ambiguous on what mapped to the event means, can we reword to something like:
```
 As of v5, context data keys prefixed with && are automatically mapped to Analytics variables and no longer include the "&&" prefix. For example, "&&products" is sent as data.__adobe.analytics.products.

Please note that these keys must be known to Analytics and are case sensitive. Find the full list of supported Analytics variables [here](https://experienceleague.adobe.com/en/docs/analytics/implementation/aep-edge/data-var-mapping). 
```